src/core_plugins/kibana/public/discover/index.html

```diff
@@ -102,6 +102,7 @@
                       class="form-control"
                       ng-model="state.interval"
                       ng-options="interval.val as interval.display for interval in intervalOptions | filter: intervalEnabled"
+                      ng-blur="toggleInterval()"
```

Comment:
One small suggestion... can we remove this function, and add two to replace it: `showInterval` and `hideInterval`? I think this explicitness clarifies the logic behind the behavior.